lib/timesheet.rb

```diff
@@ -65,12 +77,20 @@ def valid_date?
       model_citizen.valid_date?(date)
     end
 
-    def get_this_month
-      model_citizen.get_this_month
+    def get_timesheet_from_database_with_employee(month, employee)
```

Comment:
I would be worried about this kind of query in a bigger system. It's OK for your small use cases, but there are some significant performance concerns we should talk about.